packages/battery_plus/battery_plus/example/integration_test/battery_plus_test.dart

```diff
@@ -13,16 +13,20 @@ import 'package:integration_test/integration_test.dart';
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
+  final bool batteryLevelIsImplemented = Platform.isAndroid || Platform.isIOS;
+  final bool isInBatterySaveModeIsImplemented =
+      Platform.isAndroid || Platform.isIOS || Platform.isWindows;
```

Comment:
Seems that `Can get battery level` won't work on iOS Simulators, so this needs to be changed accordingly to pass the tests